### PR TITLE
[doc] Added a benchmark report on sift

### PR DESCRIPTION
**Pull request by writinwaters**
### What problem does this PR solve?

Add corresponding issue link with summary if exists -->

Issue link:

### Code changes

- [ ] Has Code change
- [ ] Has CI related scripts change

### Check List

Tests 

- [ ] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

### Note for reviewer

### CODE REVIEW

File: docs/benchmark.md [hunk 1]
```diff
@@ -1,17 +1,17 @@
 # Benchmark
 
-**Infinity** provides Python script for sift1m and gist1m dataset benchmark.
+Infinity provides a Python script for benchmarking the SIFT1M and GIST1M datasets.
 
-## Get the Infinity binary file
+## Build Infinity
```
Comment (small-turtle-1) [inline docs/benchmark.md]: Build and start.

File: docs/benchmark.md [hunk 2]
```diff
@@ -48,36 +48,39 @@ python setup.py bdist_wheel
...
+# ROUNDS indicates the number of times Python executes the benchmark, and the result represents the average duration for each run.
 
-# The following command means run benchmark with one thread, for one time using the sift dataset.
+# Perform a benchmark on the SIFT1M dataset using a single thread, running it only once.
 python remote_benchmark.py -t 1 -r 1 -d sift_1m
-
+# Perform a benchmark on the GIST1M dataset using a single thread, running it only once.
 python remote_benchmark.py -t 1 -r 1 -d gist_1m
 ```
+## A SIFT benchmark report
```
Comment (small-turtle-1) [inline docs/benchmark.md]: Add gist

File: docs/benchmark.md [hunk 3]
```diff
@@ -1,17 +1,17 @@
 # Benchmark
 
-**Infinity** provides Python script for sift1m and gist1m dataset benchmark.
+Infinity provides a Python script for benchmarking the SIFT1M and GIST1M datasets.
 
-## Get the Infinity binary file
+## Build and start Infinity
```
Comment (small-turtle-1) [inline docs/benchmark.md:5]: test

**Review by small-turtle-1 — CHANGES_REQUESTED** [review]

File: docs/benchmark.md (hunk 2 above)
Comment (writinwaters) [reply docs/benchmark.md]: skip gist for now.

File: docs/benchmark.md [hunk 4]
```diff
@@ -48,36 +48,39 @@ python setup.py bdist_wheel
...
+# Perform a benchmark on the GIST1M dataset using a single thread, running it only once.
 python remote_benchmark.py -t 1 -r 1 -d gist_1m
 ```
+## A SIFT1M Benchmark report
+
+- **Hardware**: Intel i5-12500H, 16C, 16GB
+- **Operating system**: Ubuntu 22.04
+- **Dataset**: SIFT1M; **topk**: 100; **recall**: 97%+
+- **QPS**: 10,305
+- **P99 Latency**: 0.3 ms
```
Comment (JinHai-CN) [inline docs/benchmark.md:86]: < 0.4 ms

**Review by JinHai-CN — CHANGES_REQUESTED** [review]

**Pull request merged**